createdisk.sh

```diff
@@ -308,6 +308,9 @@ ${SCP} core@api.${CRC_VM_NAME}.${BASE_DOMAIN}:/boot/ostree/rhcos-${ostree_hash}/
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -fr /etc/cni/net.d/100-crio-bridge.conf'
 ${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo rm -fr /etc/cni/net.d/200-loopback.conf'
 
+# Remove the journal logs.
+${SSH} core@api.${CRC_VM_NAME}.${BASE_DOMAIN} -- 'sudo journalctl --rotate'
```

Comment:
According to the man page, this can be combined with --vacuum-time

Reply:
@cfergeau yes I also checked the man page but when I combined with `--vacuum-time` it somehow didn't work as it should be on core os side.

```
root@crc-chc9n-master-0 core]# journalctl --rotate -m --vacuum-time=1s
Vacuuming done, freed 0B of archived journals from /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9.
[root@crc-chc9n-master-0 core]# journalctl --rotate
[root@crc-chc9n-master-0 core]# journalctl -m --vacuum-time=1s
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000001f002c-0005944ca62f3037.journal (8.0M).
Vacuuming done, freed 8.0M of archived journals from /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9.
```

Reply:
@cfergeau I tested it with the latest rc bits.

Reply:
@cfergeau Even with a clean start when I try to combine these commands, it can only clean `1.3G` but when run separately then `1.5G`. Do you know why that difference in size?

```
$ ssh -i ~/.crc/machines/crc/id_rsa -o StrictHostKeyChecking=no -o UserKnownHostsFile=/dev/null core@192.168.130.11
Warning: Permanently added '192.168.130.11' (ECDSA) to the list of known hosts.
Red Hat Enterprise Linux CoreOS 42.80.20190925.2
WARNING: Direct SSH access to machines is not recommended.

---
[systemd]
Failed Units: 1
  getty@tty1.service
[core@crc-chc9n-master-0 ~]$ sudo journalctl --rotate --vacuum-time=1s
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-000000000007c5e9-000593920647a0dc.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-000000000009bad9-00059393e8a944b3.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000000bafc8-00059395cb7a3919.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000000da4c3-00059397adb3d626.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000000f99a3-000593998fcd5fd6.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000118c56-0005939b66081357.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-000000000013813f-0005939d48a5568c.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000157549-0005939f25ef9b3c.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000176a2e-000593a1086ed7ee.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000195efa-000593a2eaefe669.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000001b53dd-000593a4cd02af07.journal (128.0M).
Vacuuming done, freed 1.3G of archived journals from /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9.

[core@crc-chc9n-master-0 ~]$ sudo journalctl --rotate
[core@crc-chc9n-master-0 ~]$ sudo journalctl --vacuum-time=1s
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-000000000007c5e9-000593920647a0dc.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-000000000009bad9-00059393e8a944b3.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000000bafc8-00059395cb7a3919.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000000da4c3-00059397adb3d626.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000000f99a3-000593998fcd5fd6.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000118c56-0005939b66081357.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-000000000013813f-0005939d48a5568c.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000157549-0005939f25ef9b3c.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000176a2e-000593a1086ed7ee.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000195efa-000593a2eaefe669.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000001b53dd-000593a4cd02af07.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000001d48cc-000593a6afa8199d.journal (120.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/user-1000@760f0062619446688a8f56f0687c8c8b-00000000001ee8d5-000593a83fd715e5.journal (8.0M).
Vacuuming done, freed 1.5G of archived journals from /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9.
```

Reply:
I assume you 'reset' the disk image between these 2 runs ? Maybe you waited more in the second run, and 200MB more of logs was generated? When it only cleaned 1.3GB, did you try to run the 2 separate commands after that to see if it cleaned up more things?

Reply:
> Maybe you waited more in the second run, and 200MB more of logs was generated? 

@cfergeau no, I tried below and again ~1.5GB clean.
```
$ sudo journalctl --rotate && sudo journalctl --vacuum-time=1s
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-000000000007c5e9-000593920647a0dc.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-000000000009bad9-00059393e8a944b3.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000000bafc8-00059395cb7a3919.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000000da4c3-00059397adb3d626.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000000f99a3-000593998fcd5fd6.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000118c56-0005939b66081357.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-000000000013813f-0005939d48a5568c.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000157549-0005939f25ef9b3c.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000176a2e-000593a1086ed7ee.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000195efa-000593a2eaefe669.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000001b53dd-000593a4cd02af07.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000001d48cc-000593a6afa8199d.journal (120.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/user-1000@760f0062619446688a8f56f0687c8c8b-00000000001ee8d5-000593a83fd715e5.journal (8.0M).
Vacuuming done, freed 1.5G of archived journals from /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9.
```

> When it only cleaned 1.3GB, did you try to run the 2 separate commands after that to see if it cleaned up more things?

@cfergeau following what I got.

```
$ sudo journalctl --rotate --vacuum-time=1s &&  sudo journalctl --rotate && sudo journalctl --vacuum-time=1s
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-000000000007c5e9-000593920647a0dc.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-000000000009bad9-00059393e8a944b3.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000000bafc8-00059395cb7a3919.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000000da4c3-00059397adb3d626.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000000f99a3-000593998fcd5fd6.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000118c56-0005939b66081357.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-000000000013813f-0005939d48a5568c.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000157549-0005939f25ef9b3c.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000176a2e-000593a1086ed7ee.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-0000000000195efa-000593a2eaefe669.journal (128.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000001b53dd-000593a4cd02af07.journal (128.0M).
Vacuuming done, freed 1.3G of archived journals from /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9.
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/system@baf83b88802c423f83b6871da1e0653c-00000000001d48cc-000593a6afa8199d.journal (120.0M).
Deleted archived journal /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9/user-1000@760f0062619446688a8f56f0687c8c8b-00000000001ee8d5-000593a83fd715e5.journal (8.0M).
Vacuuming done, freed 128.0M of archived journals from /var/log/journal/2567a02e7d8d40ef836b2e565fa8cdd9.
```

Reply:
ok, I'd add a note somewhere explaining why we run the commands separately then